src/components/dashboard/EntityStatOverview/useMonthlyUsage.ts

```diff
@@ -0,0 +1,61 @@
+import { useQuery } from '@supabase-cache-helpers/postgrest-swr';
+import { DefaultStatsWithDocument, getStatsForDashboard } from 'api/stats';
+import { isArray } from 'lodash';
+import { DateTime } from 'luxon';
+import { useMemo } from 'react';
+import { useTenantStore } from 'stores/Tenant/Store';
+import { CatalogStats_Dashboard } from 'types';
+import { hasLength } from 'utils/misc-utils';
+
+const isDefaultStatistic = (
+    datum: CatalogStats_Dashboard | DefaultStatsWithDocument
+): datum is DefaultStatsWithDocument => 'bytes_read_by_me' in datum;
```

Comment:
Can you add some comment on why this approach was taken? It was not immediately clear to me why we would check it this way and not just have the logic be more like "if capture use these variables".